packages/shared-component--postcode/src/postcode.pcss

```diff
@@ -2,9 +2,8 @@
 
 .coop-c-postcode {
   background: url("https://images.ctfassets.net/6jpeaipefazr/2oHuFOFgaPu540mDLZVRyC/bddf4e0ce16e2fdb01d27738df74627f/food-instore-2.jpeg?fm=jpg&fl=progressive&q=80&h=300&fit=crop")
-    no-repeat 0 -10px;
+    no-repeat var(--color-grey-neutral-warm) 0 -10px;
```

Comment:
I have a feeling this was a separate `background-colour` rule as some browsers didn't support `var()` when it was nestled inside another value?

Can you confirm it still works in IE, Firefox, Safari?

This might be preferable:

```
background-color: var(--color-grey-neutral-warm);
```